formulate/version.py

```diff
@@ -1,16 +1,5 @@
-# Licensed under a 3-clause BSD style license, see LICENSE.
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
-
-
-__all__ = [
-    '__version__',
-    'version',
-    'version_info',
-]
-
-
-__version__ = '0.0.8'
-version = __version__
-version_info = __version__.split('.')
+# coding: utf-8
```

Comment:
I thought this whole file should be removed since auto-generated?

Reply:
Yes, this file should be gitignored.